src/modules/Builder/api/api.types.ts

```diff
@@ -66,3 +67,24 @@ export type LorisUsersVisit = {
 export type UploadLorisUsersVisitsParams = AppletId & {
   payload: LorisUsersVisit[];
 };
+
+export type FetchIntegrationStatusParams = {
+  // currently set as snake case on the API
+  applet_id: string;
+  integration_type: IntegrationTypes;
+};
+
+export type FetchIntegrationProjectsParams = {
+  hostname: string;
+  username: string;
+  password: string;
+};
+
+export type SaveIntegrationParams = {
+  appletId: string;
+  hostname: string;
+  project: string;
+  username: string;
+  password: string;
+  integrationType: IntegrationTypes;
+};
```

Comment:
This could be refactored to better align with the BE object where `Integration` has appletId, integrationType and configuration. Where configuration is different depending on the `integrationType`.

The type for `FetchIntegrationProjectsParams` is only valid for `LorisIntegrationConfiguration`, where the `password` is optional and only used for saving the new integration.